### PR TITLE
Create en-US.md

### PR DESCRIPTION
**Pull request by litkaemily**
@mafeld @johngruen for viz

Hi @peiying2 - can you please take look at this to see if there are any editing or formatting errors?

@pmac and @alexgibson - we'll need to have this notice (when finalized) posted at https://www.mozilla.org/privacy/firefox-private-network/. I think the current notice is hard coded because I can't find it in github.

Please lmk if there are any questions!

Many thanks - emily

### CODE REVIEW

File: Firefox_Private_Network_Beta/en-US.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Version 3.0, effective July 7, 2020
+
+## At Mozilla, we design products with your privacy in mind.
+
+Firefox Private Network (FPN) protects your device’s web connections. Mozilla partners with Cloudflare to privately and securely encrypt your web traffic within Firefox.
+
+This privacy notice explains what data FPN collects, shares, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/) for how we receive, handle, and share information.
+
+#### Things you should know:
```
Comment (peiying2) [inline Firefox_Private_Network_Beta/en-US.md:11]: Lets change this heading to level 2 (##).

File: Firefox_Private_Network_Beta/en-US.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Firefox Private Network Browser Extension (Beta) Privacy Notice
+
+Version 3.0, effective July 7, 2020
```
Comment (peiying2) [inline Firefox_Private_Network_Beta/en-US.md:3]: Date format needs to include {: date... }

File: Firefox_Private_Network_Beta/en-US.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Firefox Private Network Browser Extension (Beta) Privacy Notice
```
Comment (peiying2) [inline Firefox_Private_Network_Beta/en-US.md:1]: Add the Title format

File: Firefox_Private_Network_Beta/en-US.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Firefox Private Network (FPN) protects your device’s web connections. Mozilla partners with Cloudflare to privately and securely encrypt your web traffic within Firefox.
+
+This privacy notice explains what data FPN collects, shares, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/) for how we receive, handle, and share information.
+
+#### Things you should know:
+
+### Information we collect
+
+__Firefox Account Information.__ This service requires a Firefox Account, which sends FPN your email address, locale, and IP address. Learn more about [Firefox Account data practices](https://www.mozilla.org/en-US/privacy/firefox/#firefox-accounts-join-firefox).
```
Comment (peiying2) [inline Firefox_Private_Network_Beta/en-US.md:15]: Remove `en-US` in the link.

**Comment by litkaemily** [comment]
@peiying2 Stylistic changes made. 

Yes, this is the same as https://github.com/mozilla/legal-docs/pull/1463/files?; except that https://github.com/mozilla/legal-docs/pull/1463/files? is the previous version of the product and this is the new version. Can we delete that old PR now?

File: Firefox_Private_Network_Beta/en-US.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# <span class="privacy-header-firefox">Firefox Private Network Browser Extension (Beta)</span> <span class="privacy-header-policy">Privacy Notice</span>
+
+Version 3.0, effective July 7, 2020
+{: datetime="2020-07-07" }
+
+## At Mozilla, we design products with your privacy in mind.
+
+Firefox Private Network (FPN) protects your device’s web connections. Mozilla partners with Cloudflare to privately and securely encrypt your web traffic within Firefox.
+
+This privacy notice explains what data FPN collects, shares, and why. We also adhere to the [Mozilla Privacy Policy](https://www.mozilla.org/en-US/privacy/) for how we receive, handle, and share information.
```
Comment (peiying2) [inline Firefox_Private_Network_Beta/en-US.md:10]: Missed this one.

**Comment by litkaemily** [comment]
updated

**Comment by peiying2** [comment]
> @peiying2 Stylistic changes made.
> 
> Yes, this is the same as https://github.com/mozilla/legal-docs/pull/1463/files?; except that https://github.com/mozilla/legal-docs/pull/1463/files? is the previous version of the product and this is the new version. Can we delete that old PR now?

Yes, you can remove the old PR. Your changes were not approved and thus never was pushed to production.  Let's get PR 1476 approved and pushed.

**Comment by peiying2** [comment]
@litkaemily the folder name doesn't contain Privacy Notice or PN compared to title of ToS doc.

**Pull request merged**